src/basics.md

```diff
@@ -448,6 +449,50 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-redirect-stdout-stderr-same-file]: #ex-redirect-stdout-stderr-same-file
+<a name="ex-redirect-stdout-stderr-same-file"></a>
+## Redirect both stdout and stderr of child process to the same file
+
+[![std-badge]][std] [![cat-os-badge]][cat-os]
+
+Spawns a Unix command and redirects `stdout` and `stderr` to the same
+file. It follows the same idea as [run piped external
+commands](#ex-run-piped-external-commands), however [`process::Stdio`]
+will be constructed from a file. In addition, [`File::try_clone`] is
```

Comment:
the statement might be slightly misleading. we might want to rewrite to emphasize that `process::Stdio` will write to provided `File`and that `File::try_clone` will duplicate the file handle ensuring the write pointer is synchronized for stderr and stdout